### DISCUSSION TITLE
[New Feature] Allow the left and right mouse buttons to be swapped.

### DISCUSSION THREAD

**Discussion by CCHyper** [Vinifera-Developers/Vinifera#1399] [Feature Requests]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I have attached as much information as possible *(screenshots, videos, links, etc)*.

### Description

Modern RTS games _(such as C&C3, Starcraft 2, etc)_ switch the mouse button actions so that the left button cancels input and the right button is used for issuing commands and actions. Adding an option to switch the mouse buttons to the in-game options dialog would allow the end-user to control this depending on their preference.

**Comment by Bittah** [comment]
I think that just swapping the mouse buttons and nothing else would defeat the point. While I haven't played Star Craft and haven't tried playing C&C3 right-click actions, I know that in at least Age of Empires the left mouse button is used for both selecting and canceling stuff, while the right mouse button is used for actions and this has the benefit that you won't accidentally select an object when you actually meant meant to give a move order to units that you already have selected.
